spec/controllers/application_controller_spec.rb

```diff
@@ -26,7 +26,7 @@
 
     context "current_user" do
 
-      let(:data)  { {'user_id' => '1'} }
+      let(:data)  { [{'user_id' => '1'}] }
```

Comment:
Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.